### PR TITLE
fix: labelings, titles size adjustment, removal of unnecessary sections

### PR DESCRIPTION
**Pull request by messier12**
H-4hour deadline SK rektor moment.

The unnecessary sections according to SK Rektor are only commented out lest DTKomp wants to put them back.

### CODE REVIEW

**Review by LordRonz — APPROVED** [review]
lgtm

File: lainnya/lembar-pengesahan.tex [hunk 1]
```diff
@@ -1,7 +1,8 @@
-\begin{center}
-	\large
-  \textbf{LEMBAR PENGESAHAN}
-\end{center}
+\chapter*{LEMBAR PENGESAHAN}
+%\begin{center}
+%	\large
+%  \textbf{LEMBAR PENGESAHAN}
+%\end{center}
```
Comment (threeal) [inline lainnya/lembar-pengesahan.tex:5]: pls later remove these unneeded lines on the next PR

File: main.tex [hunk 2]
```diff
@@ -108,17 +110,17 @@
   \newcommand\covercontents{sampul/konten-id.tex}
   \input{sampul/sampul-luar.tex}
 
-  % Sampul Bahasa Inggris
-  \renewcommand\covercontents{sampul/konten-en.tex}
-  \input{sampul/sampul-luar.tex}
+  % Sampul Bahasa Inggris | Tidak dibutuhkan pada format 2022
+  % \renewcommand\covercontents{sampul/konten-en.tex}
+  % \input{sampul/sampul-luar.tex}
```
Comment (threeal) [inline main.tex:115]: ditto

File: main.tex [hunk 3]
```diff
@@ -108,17 +110,17 @@
...
   % Lembar pengesahan
   \input{lainnya/lembar-pengesahan.tex}
   \newpage
 
-  % Lembar pengesahan
-  \input{lainnya/lembar-pengesahan-en.tex}
-  \newpage
+  % Lembar pengesahan | Tidak dibutuhkan pada format 2022
+  % \input{lainnya/lembar-pengesahan-en.tex}
+  % \newpage
```
Comment (threeal) [inline main.tex:123]: ditto

**Pull request merged**